Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -683,7 +745,10 @@ public IMixedRealityService GetService(Type type, string serviceName, bool showL
 
             if (service == null && showLogs)
             {
-                DebugUtilities.LogWarning($"Unable to find {serviceName} Manager.");
+                if ((ActiveProfile.LoggingLevel & LoggingLevels.Warning) != 0)
+                {
+                    Debug.LogWarning($"Unable to find {serviceName} service.");
```

Comment:
> Let's do in a separate PR

But we already made a [partial change](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3164/files#diff-f21536a45f2f15aa366accfb4e1e7d9bL686). Let's go all the way.